.github/workflows/_linux-build.yml

```diff
@@ -155,15 +156,21 @@ jobs:
           OUR_GITHUB_JOB_ID: ${{ steps.get-job-id.outputs.job-id }}
           HUGGING_FACE_HUB_TOKEN: ${{ secrets.HUGGING_FACE_HUB_TOKEN }}
         run: |
+          # Fetch aws credential from IMDs
+          eval $(python3 -c 'import boto3; import json; c=boto3.Session().get_credentials().get_frozen_credentials()._asdict(); print(f"export AWS_ACCESS_KEY_ID=\"%s\"" % c["access_key"]); print(f"export AWS_SECRET_ACCESS_KEY=\"%s\"" % c["secret_key"]); print(f"export AWS_SESSION_TOKEN=\"%s\"" % c["token"])')
```

Comment:
I feel like if were at the point where we're writing something like this we should probably just break it out into its own separate script

Reply:
Putting it in a script will force everyone to rebase when this is merged. But let me create a script ahead of time